### ISSUE TITLE
强烈怀疑作者是为了玩这个梗才做了这篇工作

### ISSUE THREAD

**Issue by wuchengyu123** [yuweihao/MambaOut#174]

**Comment by yuweihao** [comment]
That's not true, haha. I enjoy thinking about the nature of neural architectures. It's just very fitting to name the built model 'MambaOut'.

**Comment by ZihaoZheng98** [comment]
> That's not true, haha. I enjoy thinking about the nature of neural architectures. It's just very fitting to name the built model 'MambaOut'.

原来是小黑子

**Comment by yuweihao** [comment]
> > That's not true, haha. I enjoy thinking about the nature of neural architectures. It's just very fitting to name the built model 'MambaOut'.
> 
> 原来是小黑子

I am a fan of Kobe. Kobe didn't have the best talent, but he achieved greatness with the Mamba Mentality.

**Comment by zzh1161** [comment]
> > > That's not true, haha. I enjoy thinking about the nature of neural architectures. It's just very fitting to name the built model 'MambaOut'.
> > 
> > 
> > 原来是小黑子
> 
> I am a fan of Kobe. Kobe didn't have the best talent, but he achieved greatness with the Mamba Mentality.

好的，你也是曼巴门徒

**Comment by ChonghaoSima** [comment]
> > > That's not true, haha. I enjoy thinking about the nature of neural architectures. It's just very fitting to name the built model 'MambaOut'.
> > 
> > 
> > 原来是小黑子
> 
> I am a fan of Kobe. Kobe didn't have the best talent, but he achieved greatness with the Mamba Mentality.

Man, press F to pay respect. Respect to you.

**Comment by SITONGRUC** [comment]
Man！

**Comment by LyapunovStability** [comment]
> > > That's not true, haha. I enjoy thinking about the nature of neural architectures. It's just very fitting to name the built model 'MambaOut'.
> > 
> > 
> > 原来是小黑子
> 
> I am a fan of Kobe. Kobe didn't have the best talent, but he achieved greatness with the Mamba Mentality.

To be honest,  the compatibility level between "Mambaout" and this work is better than the compatibility between "Mamba" and the original work.  The author has real Mamba Mentality. Respect

**Comment by Whalesea-Gustav** [comment]
Respect Bro

**Comment by yuweihao** [comment]
> > > > That's not true, haha. I enjoy thinking about the nature of neural architectures. It's just very fitting to name the built model 'MambaOut'.
> > > 
> > > 
> > > 原来是小黑子
> > 
> > 
> > I am a fan of Kobe. Kobe didn't have the best talent, but he achieved greatness with the Mamba Mentality.
> 
> Man, press F to pay respect. Respect to you.

F

**Comment by ywh114** [comment]
Man!

**Comment by danelpeng** [comment]
> > > > > That's not true, haha. I enjoy thinking about the nature of neural architectures. It's just very fitting to name the built model 'MambaOut'.
> > > > 
> > > > 
> > > > 原来是小黑子
> > > 
> > > 
> > > I am a fan of Kobe. Kobe didn't have the best talent, but he achieved greatness with the Mamba Mentality.
> > 
> > 
> > Man, press F to pay respect. Respect to you.
> 
> F

孩子，你无敌了
